android-client-sdk/src/test/java/com/devcycle/sdk/android/api/DevCycleClientTests.kt

```diff
@@ -651,19 +656,53 @@ class DevCycleClientTests {
         val user = DevCycleClient::class.java.getDeclaredField("latestIdentifiedUser")
         user.setAccessible(true)
 
-        val client = createClient(user=DevCycleUser.builder().withIsAnonymous(true).build())
-        val callback = object: DevCycleCallback<Map<String, BaseConfigVariable>> {
-            override fun onSuccess(result: Map<String, BaseConfigVariable>) {
-                var anonUser: PopulatedUser = user.get(client) as PopulatedUser
-                verify(editor, times(1))?.putString(eq("ANONYMOUS_USER_ID"), eq(anonUser.userId))
-                // Assertions to check non-null fields
-                assertNotNull("appVersion should not be null", anonUser.appVersion)
-                assertNotNull("appBuild should not be null", anonUser.appBuild)
-                assertNotNull("language should not be null", anonUser.language)
+        val config = generateConfig("activate-flag", "Flag activated!", Variable.TypeEnum.STRING)
+        
+        mockWebServer.dispatcher = object : Dispatcher() {
+            override fun dispatch(request: RecordedRequest): MockResponse {
+                if (request.path == "/v1/events") {
+                    return MockResponse().setResponseCode(201).setBody("{\"message\": \"Success\"}")
+                } else if (request.path?.contains("/v1/mobileSDKConfig") == true) {
+                    return MockResponse().setResponseCode(200).setBody(objectMapper.writeValueAsString(config))
+                }
+                return MockResponse().setResponseCode(404)
             }
-            override fun onError(t: Throwable) {}
         }
-        client.resetUser(callback)
+
+        val client = createClient("pretend-its-a-real-sdk-key", mockWebServer.url("/").toString(), user=DevCycleUser.builder().withIsAnonymous(true).build())
```

Comment:
We should be consistent and use our test token format.
dvc_mobile_token_hash

Reply:
Also the sdk should error and fail validation on this token since it's not a valid format.